taxonomies/categories.txt

```diff
@@ -19921,7 +20043,7 @@ de:Konserven
 es:Conservas, Alimentos en conserva
 fi:purkitetut ruuat
 fr:Conserves
-hu:Konzervek, Konzerv ételek
+hu:Konzervek
```

Comment:
```suggestion
hu:Konzervek, Konzerv ételek
```

Reply:
It's already on en:canned food, cannot be there twice